src/muse/outputs/mca.py

```diff
@@ -151,50 +152,25 @@ def consumption(
     market: xr.Dataset, sectors: list[AbstractSector], **kwargs
 ) -> xr.DataArray:
     """Current consumption."""
-    from muse.outputs.sector import market_quantity
-
-    return market_quantity(market.consumption, **kwargs)
+    return market_quantity(market.consumption, **kwargs).to_dataframe().reset_index()
```

Comment:
We are not using type checking that much - if anything -, but since you're changing the return type, you should also update the signature of all of this functions to `pd.Dataframe`.